SharesCalculation.py

```diff
@@ -0,0 +1,17 @@
+import ApplicationOutput as op
+import SpouseAndParentSharesCalculation as sps
+def calculate(amount, sons, daughters, mother, father, wife):
+    if amount > 0:
+        if sons > 0 and daughters >= 0 or sons >= 0 and daughters > 0:
```

Comment:
what exactly are you doing here? Should there be any grouping, because if I am not wrong all comparison operators have the same precedence and it will go from left to right. It will check for the 1st and 2nd or (3rd) and will skip the last one because as soon as it is true it will continue further. 

Might be I am wrong, but the code is very confusing, try to simplify it a bit.

Reply:
I assemmebled all these modules to one file now, i hope these type of consequences will not occurred there.